region/brinstar/pink.json

```diff
@@ -880,11 +858,6 @@
         {
           "from": 3,
           "to": [
-            {
-              "id": 1,
-              "requires": [ "canQuickCrumbleEscape" ],
```

Comment:
Why?

Reply:
Tested it and it doesn't work. The crumble returns too quickly.

Reply:
Of course the start works, they even gave it a name: Mission Impossible.
https://crocomi.re/?r=38